src/orchestration/Orchestration.ts

```diff
@@ -135,6 +137,7 @@ export type CookieAttributes = {
 
 export type Config = {
     allowCookies: boolean;
+    allowDynamicDomEventListeners: boolean;
```

Comment:
issue: The design convention for plugins is that have their own configurations. Consider adding this configuration option to `DomEventPluginConfig` instead of the global configuration.

Reply:
Will refactor to add to `DomEventPluginConfig`